pkg/rancher-desktop/components/Help.vue

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+import { shell } from 'electron';
+import Vue from 'vue';
+import { mapState } from 'vuex';
+
+export default Vue.extend({
+  name:       'help',
+  props:      {
+    defaultUrl: {
+      type:     String,
+      default:  null,
+    },
+    tooltip: {
+      type:    String,
+      default: null,
+    },
+  },
+  computed: {
+    ...mapState('help', ['url']),
+    pageUrl(): string {
+      return this.defaultUrl ?? this.url;
+    },
+  },
+  methods:  {
+    openUrl() {
+      if (this.pageUrl) {
+        shell.openExternal(this.pageUrl);
+      }
+    },
+  },
+});
+</script>
+
+<template>
+  <div class="help-button">
+    <button
+      v-tooltip="{
+        content: pageUrl ? tooltip : null,
```

Comment:
I would move this expression to a `computed` method as well, and also map an empty string to `null`, with the assumption that an empty string tooltip still looks like a tooltip, but a `null` tooltip doesn't show up. Just guessing here though.

Reply:
Fixed.